providers/base/bin/pipewire_utils.py

```diff
@@ -452,6 +472,56 @@ def show_default_device(self, device_type):
         except subprocess.CalledProcessError as e:
             raise RuntimeError("Show default device error {}".format(repr(e)))
 
+    def _sort_and_print(self, lines: list) -> bool:
+        """
+        This method will sort the string array and print the sorted result
+
+        :param lines: the string list you would like to sort and print
+
+        :returns: is sorted and printed or not
+        """
+        if len(lines) > 0:
+            sorted_lines = sorted(lines)
+            for e in sorted_lines:
+                # logger couldn't pipe to file
+                print("{}".format(e))
+            return True
+        else:
+            return False
```

Comment:
If we just do the comparison, we can get rid of the print and simplify this part.